nwbinspector/checks/nwbfile_metadata.py

```diff
@@ -161,6 +161,18 @@ def check_subject_proper_age_range(subject: Subject):
             )
 
 
+@register_check(importance=Importance.BEST_PRACTICE_SUGGESTION, neurodata_type=Subject)
+def check_subject_age_reference(subject: Subject):
+    """Check if the Subject age reference is one of supported options."""
+    valid_options = ["birth", "gestational", None]
```

Comment:
@bendichter From discussion of https://github.com/NeurodataWithoutBorders/nwbinspector/issues/209, shouldn't `"conception"` be a valid field here as well?

Reply:
We are going with "GestationalAge" as this agrees with ontologies the DANDI schema is referencing

Reply:
https://medlineplus.gov/ency/article/002367.htm#:~:text=Gestational%20age%20is%20the%20common,37%20weeks%20are%20considered%20premature.